src/gui/gtk.h

```diff
@@ -26,6 +26,8 @@
 
 #define DT_GUI_IOP_MODULE_CONTROL_SPACING 0
 
+#define DT_GUI_THUMBSIZE_REDUCE 0.7f
```

Comment:
Ok, better...Can you just tell me why 0.7f, how this was found ? Testing ? Just curious...

Reply:
You can of course use any number less than 1 :-)
0.7 leads to thumbs with ~ half number of pixels with a perf gain of 2.
0.5 was too bad visually ...

Reply:
Thanks ! 'll test...